_blog/2023-05-08-introducing the home screen.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Welcome Home! 🏠 Introducing AntennaPod’s new default screen
+excerpt: Let's show you your new home - one of the main new features in AntennaPod 3.0.
```

Comment:
3.0 is not displayed. Maybe "Version 3.0"? That might be a bit shorter

Reply:
I increased the limit (by looking at number of characters rather than number of words).